_episodes/12-cluster.md

```diff
@@ -82,65 +58,99 @@ permissions for on the cluster.
+learn more about how to use the scheduler to submit jobs next, but for now, it can also tell us
+more information about the worker
+nodes.
+
+For example, we can view all of the worker nodes with the `{{ site.sched_info }}` command.
+
+```
+{{ site.host_prompt}} {{ site.sched_info }}
+```
+{: .bash}
+```
+{% include /snippets/12/info.snip %}
+```
 {: .output}
 
-The admin explains that Lola has to work with this remote shell session in order to
-run programs on the cluster. Launching programs that open a Graphical User Interface
-(GUI) is possible, but the interaction with the GUI will be slow as everything will
-have to get transferred through the WiFi network her laptop is currently logged into.
-Before Rob continues, he suggests to leave the cluster node again. For this, Lola can
-type in `logout` or `exit`.
+There are also specialized machines used for managing disk storage, user authentication, and other
+infrastructure-related tasks. Although we do not typically logon to or interact with these machines
+directly, they enable a number of key features like ensuring our user account and files are
+available throughout the HPC system.
 
-~~~ 
-$ logout
-~~~
-{: .language-bash}
+**This is an important point to remember: files saved on one node (computer) are often available
+everywhere on the cluster!**
 
-## Looking around more
+## What's in a node? 
 
-The admin continues to encourage Lola to look around. She explains that all of a 
-cluster's nodes have similar components as Lola's laptop or workstation. 
+All of a HPC system's nodes have the same components as your own laptop or desktop: *CPUs* (sometimes
+also called *processors* or *cores*), *memory* (or *RAM*), and *disk* space. CPUs are a computer's
+tool for actually running programs and calculations. Information about a current task is stored in
+the computer's memory. Disk is a computer's long-term storage for information it will need later.
 
-- every cluster node offers a certain amount of CPU (Central Processing Unit) cores.
-To see how many, Lola can run
-
-~~~
-$ nproc --all
-~~~
-{: .language-bash}
+> ## Explore Your Computer
+>
+> Try to find out the number of CPUs and amount of memory available on your personal computer.
+{: .challenge}
 
-- every cluster node has a certain amount of memory or 
-[RAM](https://en.wikipedia.org/wiki/Random-access_memory) (Random-access memory). 
-To see much memory `{{ site.login_host }}` in units of 
-[Gigabyte](https://en.wikipedia.org/wiki/Gigabyte) has, Lola can run
+> ## Explore The Head Node
+>
+> Now we'll compare the size of your computer with the size of the head node: To see the number of
+> processors, run:
+>
+> ```
+> {{ site.host_prompt}} nproc --all
+> ```
+> {: .language-bash}
+>
+> or
+>
+> ```
+> {{ site.host_prompt}} lscpu
+> ```
+> {: .language-bash}
+>
+> to see full details.
+> 
+> How about memory? Try running: 
+>
+> ```
+> {{ site.host_prompt}} free -m
+> ```
+> {: .language-bash}
+>
+> or for more details: 
+>
+> ```
```

Comment:
`cat /proc/meminfo` confuses people more than necessary. I suggest to put it together with `cat /proc/cpuinfo` (or `lscpu`) these into a callout box, so that there is a visual hint that this information is not required in order to proceed.

Reply:
Addressed in https://github.com/hpc-carpentry/hpc-intro/commit/8e2ca7059afd9b0080cab714be51e85c712790cf